app/townsquare/templates/townsquare/index.html

```diff
@@ -182,6 +182,18 @@
     {% include 'shared/analytics.html' %}
     {% include 'shared/footer_scripts.html' with slim=1 %}
     {% include 'shared/activity_scripts.html' %}
+
+    <!-- Send direct tip on comments -->
+    <script src="{% static "v2/js/lib/ipfs-api.js" %}"></script>
+    <script src="{% static "v2/js/ipfs.js" %}"></script>
+    <script src="{% static "v2/js/abi.js" %}"></script>
+    <script src="{% static "v2/js/amounts.js" %}"></script>
+    <script src="{% static "v2/js/lib/secrets.min.js" %}"></script>
+    <script src="{% static "v2/js/ethereumjs-accounts.js" %}"></script>
+    <script src="/dynamic/js/tokens_dynamic.js"></script>
+    <script src="{% static "onepager/js/send.js" %}"></script>
+
+    <script src="{% static "v2/js/search.js" %}"></script>
```

Comment:
This is loaded in the footer scripts today, can we rebase this pr with the last changes and ping me back?
Also review if some of the others files are already lodade into footer_scripts.html

Reply:
Ok, give 5 minutes

Reply:
I double-checked and only 'search.js' was both, here and in footer_scripts